Add aws-py-eks stack

kevanpeters opened the pull request:
Created example of `aws-py-eks` stack with all the required AWS extras. 

I see there is a PR open #509 using multilang support creating a similar example, I can rename this stack if needed as this does not use that tooling.

stack72 commented on aws-py-eks/vpc.py:
```diff
@@ -0,0 +1,104 @@
+from pulumi_aws import ec2
+
+REGION = "us-east-2"
```
We shouldn't hardcode this value

kevanpeters replied:
This is now pulling region from the config

stack72 commented on aws-py-eks/vpc.py:
```diff
@@ -0,0 +1,104 @@
+
+## Subnets
+
+vpc_subnet_1 = ec2.Subnet(
```
based on aws.get_availability_zones, we should be able to loop and create the subnets for each of the zones, rather than hardcoding

stack72 commented on aws-py-eks/vpc.py:
```diff
@@ -0,0 +1,104 @@
+    }
+)
+
+ec2.RouteTableAssociation(
```
This can be wrapped inside the loop with the availability zones

kevanpeters replied:
oh cool, I was going to use a loop but wanted to be explicit. This is exactly what I needed thanks for pointing the get_availability_zones out!

kevanpeters commented:
@stack72 Sorry for the delay on this, I was on vacation when this was reviewed. 

I incorporated the changes you called out, I am now looping through the subnets and am also no longer hard coding the region but getting it from the config. (I hope this works for tests and is a correct pattern) 

I also cleaned up some of the formatting and style to be more inline with recent additions to this repo
